conformance/lang/expressions/trap_expr.balt

```diff
@@ -0,0 +1,351 @@
+Test-Case: output
+Description: Test evaluation of expression resulting in value v,
```

Comment:
Do we need this? It's not clear what `v` is. I would sau

```suggestion
Description: Test the result of the trap expression being `e`, when evaluation of the expression completes abruptly with panic with associated value `e`.
```

Reply:
Please recheck all descriptions. We need to make the concise, but self-explanatory and clear.